functions/v2/.gitignore

```diff
@@ -0,0 +1 @@
+*.zip
```

Comment:
question: Why is this needed?
suggestion: Can it be added to the root gitignore? Scattered ignore files can be confusing.

Reply:
Currently, the GCF V2 deployment process sometimes generates extra `.zip` files.

This is a "convenience feature" to prevent us accidentally including them in commits.